Documentation/UserInterfaceRecipes.md

```diff
@@ -0,0 +1,136 @@
+# User Interface Recipes
+
+### ViewData (aka ViewModel, Presenter) Objects
```

Comment:
It sounds like there's a `ViewData` vs `ViewModel` (and `Presenter`) discussion that needs to be resolved at some point.

And is that ReactiveViewModel repo even considered useful anymore? Especially considering that the Objective-C API will be disappearing soon in the RAC 5 release?